### PR TITLE
Fix Sticker Aspect Ratio in Safari

### LINKED ISSUE #1266: Activity Feed Images Aspect Ratio broken on Safari
## Summary

Activity Feed images are stretched out of aspect ratio on Safari Browser.
### Safari
CleanShot 2022-06-21 at 12 11 56@2x

### Chrome
CleanShot 2022-06-21 at 12 21 05@2x

## Steps to reproduce

1. Go to `https://juicebox.money/#/v2/p/31`
2. Scroll to images on activity feed

## What is the _current_ bug behavior?
Images are stretched past their aspect ratio.

## What is the expected _correct_ behavior?
Safari Activity Images should look like Activity Images in Chrome.
## Relevant logs and/or screenshots
Above

## Environment

- **Device/OS**: [e.g. MacOS]
- **Browser:** [safari]

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?

fixes #1266 

### Screen shot showing, safari, chrome, firefox rendering image the same way
CleanShot 2022-07-08 at 09 05 03@2x

### Screen shot chrome production browser rendering non-sticker image
CleanShot 2022-07-08 at 09 10 36@2x
Confirming that this change doesn't break expected functionality. Confirming non-sticker images also have the same 140 x 140 image size in production.

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1362** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1362" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1362?utm_source=stack-comment).

File: src/components/RichNote.tsx [hunk 1]
```diff
@@ -66,8 +66,8 @@ export default function RichNote({
           <RichImgPreview
             src={mediaLink}
             style={{ marginTop: 10 }}
-            width="100%"
             height={140}
+            width={140}
```
Comment (aeolianeth) [inline src/components/RichNote.tsx:70]: **issue**: this compresses non-square image unnaturally.

We should find another solution here! I had a play but couldn't quite get it. Will leave it in your capable hands 🙏 

| my image | image rendered in pament memo |
| --- | --- |
|  Screen Shot 2022-07-11 at 2 03 56 PM | Screen Shot 2022-07-12 at 6 46 26 AM |

**Review by aeolianeth — CHANGES_REQUESTED** [review]
Thanks @benschac ! There's one issue I found: https://github.com/jbx-protocol/juice-interface/pull/1362/files#r918365829

🏓

**Comment by tomquirk** [comment]
@benschac frendly bump here.

**Comment by benschac** [comment]
I accidentally synced changed in my code editor and pulled in changes from the last two weeks. Going to close this PR and open a new one since the code change is tiny.

**Pull request closed without merging**